src/test/java/com/jcabi/github/IssueTest.java

```diff
@@ -188,6 +188,9 @@ private Repo repo() {
         Mockito.doReturn(coords).when(repo).coordinates();
         Mockito.doReturn("user").when(coords).user();
         Mockito.doReturn("repo").when(coords).repo();
+        final Github github = Mockito.mock(Github.class);
+        Mockito.doReturn(github).when(repo).github();
+        Mockito.doReturn(Constants.ROOT_REPO).when(github).rootRepoPath();
```

Comment:
@kkashi01 See, the first malignant effect of the method on the Github interface: now you have to mock it throughout all the unit tests.